presets/workspace/inference/vllm/inference_api.py

```diff
@@ -79,16 +81,32 @@ def parse_args(self, *args, **kwargs):
         kaito_args = args[0]
         runtime_args = args[1] # Remaining args
 
+        kv_cache_cpu_memory_utilization = 0.5
         # Load KAITO config
         if kaito_args.kaito_config_file:
             file_config = KaitoConfig.from_yaml(kaito_args.kaito_config_file)
             if kaito_args.kaito_max_probe_steps is None:
                 kaito_args.kaito_max_probe_steps = file_config.max_probe_steps
+            if file_config.kv_cache_cpu_memory_utilization is not None:
+                kv_cache_cpu_memory_utilization = file_config.kv_cache_cpu_memory_utilization
 
             for key, value in file_config.vllm.items():
                 runtime_args.append(f"--{key}")
                 runtime_args.append(str(value))
 
+        if envs.VLLM_USE_V1 and kv_cache_cpu_memory_utilization > 0.001:
```

Comment:
This check might not take effect. This is the env passed from user side. 
However, vllm will determine v0 or v1 based on hardware again. even if you use T4 with VLLM_USE_V1=1, vllm will fallback to v0.
The reliable check should be happened after `engine_args.create_engine_config()`. 

Let's check it with T4 to see if fallback is successful.